FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.DataWriter.UnitTests/Azure/AzureBlobContainerClientTests.cs

```diff
@@ -523,9 +526,187 @@ public async void RenewLease_WhenBlobNotExists_NoLeaseShouldReturn()
             await blobContainerClient.DeleteIfExistsAsync();
         }
 
+        [SkippableFact]
+        public async Task GivenANonExistDirectoryName_WhenDeleteIfExists_NoExceptionShouldBeThrown()
+        {
+            var uniqueContainerName = Guid.NewGuid().ToString("N");
+            AzureBlobContainerClient adlsClient = GetTestAdlsGen2Client(uniqueContainerName);
+            Skip.If(adlsClient == null);
+
+            var blobContainerClient = GetBlobContainerClient(uniqueContainerName);
+            blobContainerClient.CreateIfNotExists();
+
+            try
+            {
+                var exception = await Record.ExceptionAsync(async () => await adlsClient.DeleteDirectoryIfExistsAsync("foldernotexist"));
+                Assert.Null(exception);
+            }
+            finally
+            {
+                await blobContainerClient.DeleteAsync();
+            }
+        }
+
+        [SkippableFact]
+        public async Task GivenANonExistDirectoryName_WhenDeleteIfExists_DirectoryAndBlobsShouldBeDeleted()
```

Comment:
Seems should be GivenAnExistDirectoryName_WhenDeleteIfExists_DirectoryAndBlobsShouldBeDeleted()?

Reply:
Thanks. Fixed